app/controllers/foreman_tasks/api/tasks_controller.rb

```diff
@@ -76,6 +76,36 @@ def bulk_search
         render :json => ret
       end
 
+      api :PUT, '/tasks/bulk_resume', 'Resume all paused error tasks'
+      def bulk_resume
+        scope = resource_scope_for_index.select('DISTINCT foreman_tasks_tasks.*')
+        scope = scope.where(:state => :paused)
+        scope = scope.where(:result => :error)
+
+        resumed = []
+        failed = []
+        skipped = []
+        scope.all.each do |task|
```

Comment:
we should probably use find_each to avoid loading all the tasks at once into the memmory http://apidock.com/rails/ActiveRecord/Batches/ClassMethods/find_each